docs/docs/configuration/settings.md

```diff
@@ -43,6 +43,8 @@ The configuration file, `configuration.json` is in the Oni2 directory, whose loc
 
 - `editor.largeFileOptimizations` __(_bool_ default: `true`)__ - When `true`, Onivim will turn off certain settings like syntax highlighting for large files.
 
+- `editor.enablePreview` __(_bool_ default: `true`)__ - When `true`, Onivim will open files in _preview mode_ unless a change is made or the tab is double-clicked. In _preview mode_, the editor tab will be re-used.
```

Comment:
We should switch this to `workbench.editor.enablePreview` to be consistent with VSCode

Reply:
Saw this was fixed in latest - thanks!